Sources/Networking/WebBillingHTTPRequestPath.swift

```diff
@@ -50,12 +54,20 @@ extension HTTPRequest.WebBillingPath: HTTPRequestPath {
         switch self {
         case let .getWebOfferingProducts(appUserID):
             return "/rcbilling/v1/subscribers/\(appUserID.trimmedAndEscaped)/offering_products"
+        case let .getWebBillingProducts(userId, productIds):
+            let encodedUserId = userId.trimmedAndEscaped
+            let encodedProductIds = productIds.sorted().map { productId in
```

Comment:
Hmm I guess the sorting is to make tests deterministic? Is there some data structure that can be used as a "sorted set" of sorts? (kotlin does have a data structure for that FWIW). If not, I think this is probably ok for now